FPBinarySearchTreeTord/src/js/BinarySearchTree.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+export function createTree(inputArray){
+  if(inputArray.length != 0){
+    var returnArray = inputArrayInTree(inputArray,0,[]);
+  }
+  else{
+    var returnArray = [];
+  }
+  return returnArray;
+}
+
+export function inputArrayInTree(inputArray,placeInInputArray,returnArray){
+    if(inputArray.length-1 >= placeInInputArray){
+      returnArray = findPlaceForInput(returnArray,0,inputArray[placeInInputArray]);
+      returnArray = inputArrayInTree(inputArray,(placeInInputArray+1),returnArray);
+    }
+  return returnArray;
+}
+
```

Comment:
Misleading name. Perhaps think about the structure of these functions a bit more.